xgboost_ray/main.py

```diff
@@ -356,6 +412,7 @@ def train(params: Dict,
           gpus_per_actor: int = -1,
           resources_per_actor: Optional[Dict] = None,
           max_actor_restarts: int = 0,
+          tune: bool = False,
```

Comment:
Let's decouple this - I'll address this in the main feedback

Reply:
Done!